### PR TITLE
MPL exponenential property additive offset

### PR DESCRIPTION
**Pull request by endJunction**
A small generalization.

Can be used for exponential saturation-head relation as described by Tracy2006 paper for analytical solutions of Richards equation. [https://doi.org/10.1029/2005WR004638]

1. [ ] Feature description was added to the [changelog](https://github.com/ufz/ogs/wiki/Release-notes-6.3.1)
2. [x] Tests covering your feature were added?
3. [x] Any new feature or behavior change was documented?

### CODE REVIEW

**Comment by TomFischer** [comment]
The test 'Tests/Data/RichardsMechanics/deformation_dependent_porosity_swelling.prj' needs an update in the exponential property specification.

**Comment by TomFischer** [comment]
Please, can you motivate a little bit the changes. The offset in the existing tests is zero and thus the changes aren't really necessary.

File: MaterialLib/MPL/Properties/CreateExponential.cpp [hunk 1]
```diff
@@ -43,6 +43,9 @@ std::unique_ptr<Exponential> createExponential(
     auto const factor =
         //! \ogs_file_param{properties__property__Exponential__exponent__factor}
         exponent_data_config.getConfigParameter<double>("factor");
+    auto const offset =
```
Comment (TomFischer) [inline MaterialLib/MPL/Properties/CreateExponential.cpp:46]: The offset should not be in the exponent specification.

**Comment by endJunction** [comment]
> Please, can you motivate a little bit the changes. The offset in the existing tests is zero and thus the changes aren't really necessary.
Updated the PR description.

File: MaterialLib/MPL/Properties/CreateExponential.cpp (hunk 1 above)
Comment (endJunction) [reply MaterialLib/MPL/Properties/CreateExponential.cpp:47]: Thanks! Fixed. (I hope ;)

**Comment by TomFischer** [comment]
The HT faulted cube project file needs an update. There are two media definitions. The offset is introduced only in one medium definition.
```
diff --git a/Tests/Data/Parabolic/HT/FaultedCube/Ra_795_fault_bcgs_jacobi.prj b/Tests/Data/Parabolic/HT/FaultedCube/Ra_795_fault_bcgs_jacobi.prj
index f021231de..4cd58edc5 100644
--- a/Tests/Data/Parabolic/HT/FaultedCube/Ra_795_fault_bcgs_jacobi.prj
+++ b/Tests/Data/Parabolic/HT/FaultedCube/Ra_795_fault_bcgs_jacobi.prj
@@ -140,6 +140,7 @@
                         <property>
                             <name>viscosity</name>
                             <type>Exponential</type>
+                            <offset>0</offset>
                             <reference_value>1.0e-3</reference_value>
                             <exponent>
                                 <variable_name>temperature</variable_name>
```

**Comment by TomFischer** [comment]
> > Please, can you motivate a little bit the changes. The offset in the existing tests is zero and thus the changes aren't really necessary.
> 
> Updated the PR description.

Thanks!

**Pull request merged**